app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class AuthenticationServiceTest {
+
+    private AuthenticationService authenticationService;
+    private static final String SECRET ="12345678901234567890123456789010";
+    private static final String VALID_TOKEN= "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGw";
+    private static final String INVALID_TOKEN= "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGG0";
```

Comment:
```suggestion
    private static final String INVALID_TOKEN = VALID_TOKEN + "WRONG";
```

종립님이 말씀해주신 것처럼 이렇게 해보는 것도 좋을 것 같습니다